tests/unit/workflows/test_workflows_tasks_arxiv.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+from six import StringIO
+
+from inspire_schemas.utils import load_schema
+from inspirehep.dojson.utils import validate
+from inspirehep.modules.workflows.tasks.arxiv import (
+    arxiv_derive_inspire_categories
+)
+
+
+class StubObj(object):
+    def __init__(self, data, extra_data):
+        self.data = data
+        self.extra_data = extra_data
+
+
+class DummyEng(object):
+    pass
+
+
+def test_arxiv_derive_inspire_categories():
+    schema = load_schema('hep')
+    subschema = schema['properties']['inspire_categories']
+
+    obj = StubObj(
+        {
+            'arxiv_eprints': [
+                {
+                    'categories': [
+                        'alg-geom', # with a corresponding inspire_category
+                        'q-bio', # without a corresponding inspire_category
```

Comment:
Make sense! I have done in this way because 1 can be a bug but 3...
We have 3 categories "mapped" in this way:

| arxiv_categ | inspire_categ |
| ------------- | ------------- |
| q-bio  | None  |
| q-fin  | None  |
| stat | None |